ui/packages/lib/src/providers/project/currentProjectContext.js

```diff
@@ -2,10 +2,34 @@ import React, { useContext, useMemo } from "react";
 import ProjectsContext from "./projectsContext";
 
 const CurrentProjectContext = React.createContext({});
+const projectIdKey = "lastSelectedProjectId";
+
+const getSelectedProjectId = (projectId, projects) => {
+  let selectedProjectId = projectId;
+  if (selectedProjectId !== undefined) {
+    localStorage.setItem(projectIdKey, selectedProjectId);
+    return selectedProjectId;
+  }
```

Comment:
I don't think `selectedProjectId` variable necessary here. We can just use `projectId`.

```
  if (projectId !== undefined) {
    localStorage.setItem(projectIdKey, projectId);
    return projectId;
  }
```

Reply:
updated